source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h

```diff
@@ -30,6 +30,6 @@ using namespace AdaptiveCards;
                                       toView:(UIView *)view;
 // find date and time string, and replace them in NSDateFormatterCompactStyle, NSDateFormatterMediumStyle or
 // NSDateFormatterLongStyle of local language
-+ (std::string) getLocalizedDate:(std::shared_ptr<TextBlock> const &)txtBlck;
```

Comment:
missplaced the second parameter name.
+ (std::string) getLocalizedDate:(std::string const &)text  language:(std::string const &)language;